app/views/handlers/view_submitted_response_pdf.py

```diff
@@ -54,7 +54,10 @@ def __init__(
     @property
     def filename(self) -> str:
         """The name to use for the PDF file"""
-        return f"{self._metadata['schema_name']}.pdf"
+        if self._questionnaire_store.submitted_at:
+            return f"{self._schema.json['title']}-{self._questionnaire_store.submitted_at.date()}.pdf".replace(
+                " ", "-"
+            ).lower()
```

Comment:
I think we want to add another replace, so chain them and avoid underscores in file name (if I understand the card correctly), so:
```suggestion
            ).replace("_", "-").lower()
```